server/src/main/java/org/elasticsearch/action/admin/cluster/snapshots/create/CreateSnapshotRequest.java

```diff
@@ -482,6 +508,7 @@ public String toString() {
             ", includeGlobalState=" + includeGlobalState +
             ", waitForCompletion=" + waitForCompletion +
             ", masterNodeTimeout=" + masterNodeTimeout +
+            ", _meta=" + userMetadata +
```

Comment:
I don't think you need the underscore here, users can't add any other top level fields here anyway so no need to make this look "more internal" than the other fields?

Reply:
Good point, I'll change this.

Reply:
Probably to follow the convention we have with the mapping `_meta` field? https://www.elastic.co/guide/en/elasticsearch/reference/7.x/mapping-meta-field.html